src/bundle/ui-dev/src/modules/content-tree/components/popup-actions/popup.actions.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect, useRef } from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+import Icon from '../../../common/icon/icon';
+
+const PopupActions = ({ listRef, options }) => {
+    const containerRef = useRef();
+    const containerItemsRef = useRef();
+    const [isExpanded, setIsExpanded] = useState(false);
+    const toggleExpanded = () => {
+        setIsExpanded((prevState) => !prevState);
+    };
+    const getHeaderActions = () => {
+        const { headerActions } = window.eZ.adminUiConfig.contentTreeWidget;
+        const headerActionsArr = headerActions ? [...headerActions] : [];
+
+        return headerActionsArr.sort((headerActionA, headerActionB) => {
+            return headerActionB.priority - headerActionA.priority;
+        });
+    }
+    const renderItem = (item) => {
+        const Component = item.component;
+
+        return (
+            <li
+                class="c-popup-actions__item"
+                key={item.id}
+                onClick={() => {
```

Comment:
Maybe just ?
```suggestion
                onClick={() => setIsExpanded((prevState) => !prevState) }
```

Reply:
we can do:
```suggestion
                onClick={toggleExpanded}
```
which for me would be the clearest solution.